ci/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+git fetch origin master:master
+git diff --name-only master $HEAD > modified
+echo "these files have changed"
+cat modified
```

Comment:
We wouldn't need to create a temporary file if we reorder the commends:
```diff
-git diff --name-only master $HEAD > modified
 echo "these files have changed"
+git diff --name-only master $HEAD
-cat modified
```

Reply:
Ah yes, good point. Thanks for pointing out. We could even possible log the file names by piping to echo.